src/App.js

```diff
@@ -1,5 +1,4 @@
 import "./App.css";
-
 import { Footer } from "./components/Footer/Footer";
 import { NavBar } from "./components/NavBar/NavBar";
 import { RoutersPath } from "./Routers/RoutersPath";
```

Comment:
You can use an `index.js` file and add all your exports here and you can simply import all your component from a single file.